app/src/main/java/com/example/newsportal/data/local/NewsDao.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.newsportal.data.local
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.example.newsportal.data.local.model.ArticleLocal
+
+@Dao
+interface NewsDao {
+
+    @Query("SELECT * from articlelocal")
+    fun findAll(): LiveData<List<ArticleLocal>>

```

Comment:
it need to be replaced with coroutines flow in the future, live data is primary designed to communicate between view and viewmodel, not for data layer